test/common/test_common.cpp

```diff
@@ -173,17 +173,38 @@ TEST (PCL, Eigen)
 }
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-TEST (PCL, PointCloud)
-{
-  PointCloud<PointXYZ> cloud;
-  cloud.width = 640;
-  cloud.height = 480;
+struct pointCloudTest : public testing::Test {
+  protected:
+    virtual void SetUp() {
+      cloud.width = 640;
```

Comment:
This is a bad since the cloud.size and cloud.width*cloud.height don't match. It'll be better to create a proper cloud

Eg: After inserting 5 objects, cloud.width is 5, not 640*480 + 5

Reply:
I am not sure what's bad about it? Can you elaborate more?

Reply:
Exactly. By having size != width*height, the cloud is in indeterminate state.